src/utility/mapping.js

```diff
@@ -8,16 +9,24 @@ import {
 
 const originalSuffix = '_$';
 
+/*
+There is a lot of cloning here to handle mapping.
+When implementing nesting we need to use cloneDeep.
+This method should be optimized if possible as it
+would be slow on large datasets. Test performance.
+*/
 const mapItem = (schema, item) => {
   /* eslint no-param-reassign:"off" */
   each(schema, (field) => {
-    item[`${field.name}${originalSuffix}`] = item[field.name];
+    item[`${field.name}${originalSuffix}`] = clone(item[field.name]);
   });
 
   each(schema, (field) => {
     if (field.mapName) {
-      item[field.mapName] = item[`${field.name}${originalSuffix}`];
+      item[field.mapName] = clone(item[`${field.name}${originalSuffix}`]);
```

Comment:
Shouldn't `item[field.name]` also be deleted in case it's mapped?
From what I see, and if I'm using it correctly, we still get duplicate values after mapping (before this PR there were triple :) )

<img width="858" alt="screen shot 2018-09-03 at 08 53 52" src="https://user-images.githubusercontent.com/22771787/44971716-ef33d580-af56-11e8-875b-00b8c2b44a8b.png">
<img width="931" alt="screen shot 2018-09-03 at 08 54 02" src="https://user-images.githubusercontent.com/22771787/44971720-f6f37a00-af56-11e8-9c23-d6a6c22901c9.png">

Reply:
Hm, according to test no. 😄 
I test we have situation where we have three columns, two are mapped to each other and one is not mapped. If I delete `item[field.name]` there would be no `active` property from tests.

How did you test this? SDK should be updated in chameleon-vuetify and then used in Builder.

Reply:
Oh, sorry, I get what you mean.
Just a second. :)